packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css

```diff
@@ -72,14 +76,14 @@
     }
```

Comment:
Also, make the font-size for mobile devices bigger since now the font is too small and we can cover more space due to the increase of the animation space.